### PR TITLE
Add MBS 0123 and MBS 0173

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
New MBS questionnaires.

### How to review
Check against schema definitions in slides

### CODE REVIEW

File: data/en/mbs_0123.json [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+                            "content": [{
+                                "title": "",
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Commission and fees",
+                                    "content": [{
+                                        "description": "Include:",
+                                        "list": [
+                                            "For example, as a travel agent or tour operator, where you do not hold title to goods/services."
```
Comment (charlotte-hirst) [inline data/en/mbs_0123.json]: this shouldnt have an include list the slides are wrong, its an example it doesnt need to be on it

File: data/en/mbs_0123.json [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+                                }
+                            ]
+                        }]
+                    },
+                    {
+                        "id": "commission-and-fees-block",
+                        "type": "Question",
+                        "title": "Commission and fees",
+                        "questions": [{
+                            "guidance": {
```
Comment (charlotte-hirst) [inline data/en/mbs_0123.json]: this shouldnt be in a blue block as its not an include/exclude

File: data/en/mbs_0123.json [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "sales-on-own-account-block",
+                        "type": "Question",
+                        "title": "Sales on own account and turnover from other activities",
+                        "questions": [{
+                            "guidance": {
+                                "content": [{
```
Comment (charlotte-hirst) [inline data/en/mbs_0123.json]: example need to come out of blue panel as guidance not include/exclude

File: data/en/mbs_0173.json [hunk 4]
```diff
@@ -0,0 +1,601 @@
...
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Commission and fees",
+                                    "content": [{
+                                        "description": "Include:",
+                                        "list": [
+                                            "For example, as a travel agent or tour operator, where you do not hold title to goods/services."
```
Comment (charlotte-hirst) [inline data/en/mbs_0173.json]: remove as an example doesnt need to be here

File: data/en/mbs_0173.json [hunk 5]
```diff
@@ -0,0 +1,601 @@
...
+                        }]
+                    },
+                    {
+                        "id": "commission-and-fees-block",
+                        "type": "Question",
+                        "title": "Commission and fees",
+                        "questions": [{
+                            "guidance": {
+                                "content": [{
+                                    "description": "For example, as a travel agent, where you do not hold title to goods/services"
```
Comment (charlotte-hirst) [inline data/en/mbs_0173.json]: remove from blue panel and move above

File: data/en/mbs_0173.json [hunk 6]
```diff
@@ -0,0 +1,601 @@
...
+                                "mandatory": true
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "sales-on-own-account-block",
+                        "type": "Question",
+                        "title": "Sales on own account and turnover from other activities",
+                        "questions": [{
+                            "description": "For example, as a tour operator",
```
Comment (charlotte-hirst) [inline data/en/mbs_0173.json]: same remove from blue panel to above it

Comment (charlotte-hirst) [reply data/en/mbs_0173.json]: ignore this one is correct

File: data/en/mbs_0173.json [hunk 7]
```diff
@@ -0,0 +1,601 @@
...
+                    },
+
+                    {
+                        "id": "employees-block",
+                        "type": "Question",
+                        "title": "Employees",
+                        "questions": [{
+                            "guidance": {
+                                "content": [{
+                                        "description": "An employee is anyone aged 16 years or over that your organisation directly pays from its payroll(s), in return for carrying out a full-time or part-time job or being on a training scheme."
```
Comment (charlotte-hirst) [inline data/en/mbs_0173.json]: needs to be above blue panel

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

File: data/en/mbs_0173.json [hunk 8]
```diff
@@ -0,0 +1,586 @@
...
+                                            "voluntary workers",
+                                            "former employees only receiving a pension",
+                                            "self-employed workers"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "employees-question",
+                            "type": "General",
+                            "title": "On {{exercise.start_date|format_date}}, what was the number of employees for {{respondent.trad_as_or_ru_name}}?",
```
Comment (shahi645) [inline data/en/mbs_0173.json]: Needs to be `employment_date` not start date

**Review by shahi645 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0173.json [hunk 9]
```diff
@@ -0,0 +1,586 @@
...
+                                    },
+                                    {
+                                        "label": "No I need to change this",
+                                        "value": "No"
+                                    }
+                                ],
+                                "mandatory": true
+                            }],
+                            "id": "confirm-zero-employees-question",
+                            "title": "On {{exercise.start_date|format_date}}, the number of employees for {{respondent.trad_as_or_ru_name}} was <em>0</em>, is this correct?"
```
Comment (shahi645) [inline data/en/mbs_0173.json]: employment date

File: data/en/mbs_0173.json [hunk 10]
```diff
@@ -0,0 +1,586 @@
...
+                                            "voluntary workers",
+                                            "former employees only receiving a pension",
+                                            "self-employed workers"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "number-of-employees-split-question",
+                            "title": "Of the <em>{{answers['number-of-employees-total']}}</em> total employees employed on {{exercise.start_date|format_date}}, how many male and female employees worked the following hours?",
+                            "type": "Calculated"
```
Comment (shahi645) [inline data/en/mbs_0173.json]: employment date

**Review by shahi645 — CHANGES_REQUESTED** [review]

File: data/en/mbs_0173.json [hunk 11]
```diff
@@ -0,0 +1,586 @@
...
+                                    "when": [{
+                                        "value": "No",
+                                        "id": "confirm-zero-employees-answer",
+                                        "condition": "equals"
+                                    }]
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "number-of-employees-split-block"
```
Comment (shahi645) [inline data/en/mbs_0173.json]: wrong routing rule, needs to go to significant changes

**Review by shahi645 — CHANGES_REQUESTED** [review]
Sorry last one!

File: data/en/mbs_0173.json (hunk 11 above)
Comment (MIslam92) [reply data/en/mbs_0173.json]: 🙃

File: data/en/mbs_0173.json [hunk 12]
```diff
@@ -72,46 +58,29 @@
                                             "special/calendar events",
                                             "weather",
                                             "price effects",
-                                            "currency effects (increase/decrease in the currency value)"
+                                            "currently effects (increase/decrease in the currency value)"
```
Comment (shahi645) [inline data/en/mbs_0173.json]: currency 😆

**Review by shahi645 — CHANGES_REQUESTED** [review]
Also u are changing file name instead of adding a new one, so will overwrites 0255

**Pull request closed without merging**